logconsumer_test.go

```diff
@@ -523,10 +532,10 @@ func (l *bufLogger) String() string {
 func Test_MultiContainerLogConsumer_CancelledContext(t *testing.T) {
 	// Capture global logger.
 	logger := &bufLogger{}
-	Logger = logger
-	oldLogger := Logger
+	oldLogger := logging.Logger
+	SetLogger(logger)
 	t.Cleanup(func() {
-		Logger = oldLogger
+		logging.Logger = oldLogger
 	})
```

Comment:
Please check, it felt like the order of operations was wrong here (original first patched the logger, then it stored it as oldLogger, with the clean up not really changing anything?